index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#welcome-section">Contact</a
+          >
+        </li>
+        <li>
+          <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#projects"
+            >Work</a
+          >
+        </li>
+        <li>
+          <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#contact"
+            >About</a
+          >
+        </li>
```

Comment:
Append the list <li> child properly in the <ul> parent element.

Reply:
Please kindly make changes to the comment above.

Reply:
- [ ] Your comment is polite 👍🏿 please always add  why a change is requested in  your comments